Makefile

```diff
@@ -1,18 +1,24 @@
+# NOTE: temporary hand edited to add --version-prefix.
+
 # WARNING: This file is autogenerated - changes will be overwritten if not made via https://github.com/pulumi/ci-mgmt
 
 PACK := tls
 ORG := pulumi
 PROJECT := github.com/$(ORG)/pulumi-$(PACK)
-PROVIDER_PATH := provider/v4
+PROVIDER_PATH := provider/v5
 VERSION_PATH := $(PROVIDER_PATH)/pkg/version.Version
 TFGEN := pulumi-tfgen-$(PACK)
 PROVIDER := pulumi-resource-$(PACK)
-VERSION := $(shell pulumictl get version)
+VERSION := $(shell pulumictl get version --version-prefix 5.0.0)
```

Comment:
If I run this locally, I get the correct behavior without this change.

```sh
$ git tag v5.0.0
$ pulumictl get version --language generic
5.0.0
```

That should cover releases produced via tags. I tried this with a few different tags as well.

Reply:
Yes to get this behavior in CI you need to publish the tag, which might trigger a release :) And if CI-local fidelity is lost then some tests break.